prow/test/integration/test/sub_test.go

```diff
@@ -476,7 +476,7 @@ this-is-from-repo5
 			// log as much as possible to make it easier to see why a test
 			// failed), or when the test succeeds (where we clean up the ProwJob
 			// that was created by sub).
-			timeout := 90 * time.Second
+			timeout := 120 * time.Second
```

Comment:
If we need to raise this to > 300 seconds (or some other upper limit we think crosses the threshold of what is "acceptable"), let's revisit the idea of optimizing the performance of these tests.